MantidPlot/src/main.cpp

```diff
@@ -35,7 +35,6 @@
 #include <QSplashScreen>
 #include <QThread>
 #include <QTimer>
-#include <iostream>
```

Comment:
I think this file will also need to be left alone as it is the `main` routine for MantidPlot and at this point the mantid logging framework as not yet been set up.